Checkout.podspec

```diff
@@ -12,11 +12,11 @@ TODO: Add long description of the pod here.
   s.author = { "Checkout.com Integration" => "integration@checkout.com" }
   s.source       = { :git => "https://github.com/checkout/frames-ios.git", :tag => s.version }
 
-  s.ios.deployment_target = '10.0'
-  s.swift_version = "5.0"
+  s.ios.deployment_target = '11.0'
```

Comment:
Fair point, our test project is using iOS 12. 

The reason I used iOS 11 and not 12 is that Xcode 14 still supports iOS 11, but it no longer supports iOS 10 ([Xcode 14 Release notes](https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes). Since Xcode 14 is now required to push to iTunesConnect, we can safely remove support from iOS 10, as by analogy the store doesn't support it.

Dropping support for iOS 11 however would be more of a Product decision, as technically we don't need to do anything  (that we know of) to support it and it is supported by the store (and therefore maybe some of our clients).